tests/backups/info-logs/test.log

```diff
@@ -0,0 +1,276 @@
+../raw_data/71859-71861-71863.json
```

Comment:
Was this log file intentionally included? It looks like an attempt was made to ignore log files above.